src/main/java/duke/logic/Parser.java

```diff
@@ -0,0 +1,275 @@
+package duke.logic;
+
+import duke.command.*;
+import duke.exception.*;
```

Comment:
Should this be expanded out to explicit imports?
I noticed this in a few other places too

Reply:
I am not opposed to the used of explicit imports. However, as the class will be using the entire `duke.command` subpackage, I have decided it is much cleaner to use wildcard import.